builder/amazon/common/ami_config_test.go

```diff
@@ -30,6 +30,13 @@ func TestAMIConfigPrepare_regions(t *testing.T) {
 		t.Fatalf("shouldn't have err: %s", err)
 	}
 
+	c.AMIRegions = []string{"ap-northeast-1", "ap-northeast-2", "ap-southeast-1",
```

Comment:
It looks like this should be the same as the `regions` list below. If so, we should change this to a function e.g. `func listEC2Regions() [12]string` so we don't need to keep this list up-to-date in two places.